java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -695,13 +704,16 @@ private Description updateCode(
     }
 
     if (update) {
-      Description.Builder builder = buildDescription(tree);
-      SuggestedFix.Builder fixBuilder = SuggestedFix.builder();
-      fixBuilder.replace(expr, replacementString);
+      SuggestedFix.Builder fixBuilder =
+          Optional.ofNullable(
+                  ASTHelpers.findEnclosingNode(state.getPath(), ExpressionStatementTree.class))
+              .filter(
```

Comment:
When we'll have more such API patterns to support we can extract out these lines.